.github/workflows/deploy.yml

```
@@ -77,7 +77,44 @@ jobs:
         with:
           repository: AdobeDocs/adp-devsite-scripts
           path: scripts
-          ref: main
+          ref: deploy-fail
```

is this also left over from testing?

```suggestion
          ref: main
```